src/Form/GraphQLQueryMapForm.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+namespace Drupal\graphql\Form;
+
+use Drupal\Core\Entity\EntityForm;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Form controller for GraphQL query map forms.
+ */
+class GraphQLQueryMapForm extends EntityForm {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function form(array $form, FormStateInterface $form_state) {
+    $form = parent::form($form, $form_state);
+
+    $query_map = $this->entity;
+
+    $form['#title'] = $this->t('Edit %title GraphQL Query Map', ['%title' => $query_map->id()]);
+
+    $form['version_id'] = [
```

Comment:
Yeah... I was thinking about that... but what if you want to make some small adjustments without modifying the version id? Or does that not make sense?